python_src/src/lamp_py/aws/s3.py

```diff
@@ -376,7 +376,7 @@ def write_parquet_file(
 
     # call the visitor function if it exists
     if visitor_func is not None:
-        visitor_func(write_path)
+        visitor_func(123)
```

Comment:
```suggestion
        visitor_func(write_path)
```

Reply:
yeah. i accidentally commited that after running black